en/08_Changelogs/6.0.0.md

```diff
@@ -181,10 +183,25 @@ SilverStripe\UserForms\Model\UserDefinedForm:
 
 </details>
 
-### Other new features
+### Changes to the templating/view layer {#view-layer}
```

Comment:
"Other new features" renamed and is now below the new section.

Giving this section a nice broad name because I'll use this section for the overall changelog info for the template layer refactor.